src/Symfony/Component/EventDispatcher/Tests/Debug/TraceableEventDispatcherTest.php

```diff
@@ -124,6 +124,40 @@ public function testGetCalledListeners()
         $this->assertEquals(array(), $tdispatcher->getNotCalledListeners());
     }
 
+    public function testGetOrphanedEvents()
+    {
+        // SCENARIO - Empty when created
+
+        // GIVEN
+        $tdispatcher = new TraceableEventDispatcher(new EventDispatcher(), new Stopwatch());
+        // WHEN
+        $events = $tdispatcher->getOrphanedEvents();
+        // THEN
+        $this->assertEmpty($events);
+
+        // SCENARIO - Returns orphaned event
```

Comment:
different scenarios should be different tests (so that a failure in the first oen does not prevent others from running)